webrtc/src/track/track_local/track_local_static_rtp.rs

```diff
@@ -42,6 +44,83 @@ impl TrackLocalStaticRTP {
             .iter()
             .all(|b| b.sender_paused.load(Ordering::SeqCst))
     }
+
+    /// write_rtp_with_extensions writes a RTP Packet to the TrackLocalStaticRTP
+    /// If one PeerConnection fails the packets will still be sent to
+    /// all PeerConnections. The error message will contain the ID of the failed
+    /// PeerConnections so you can remove them
+    ///
+    /// If the RTCRtpSender direction is such that no packets should be sent, any call to this
+    /// function are blocked internally. Care must be taken to not increase the sequence number
+    /// while the sender is paused. While the actual _sending_ is blocked, the receiver will
+    /// miss out when the sequence number "rolls over", which in turn will break SRTP.
+    ///
+    /// Extensions that are already configured on the packet are overwritten by extensions in
+    /// `extensions`.
+    pub async fn write_rtp_with_extensions(
+        &self,
+        p: rtp::packet::Packet,
+        extensions: &[rtp::extension::HeaderExtension],
+    ) -> Result<usize> {
```

Comment:
Why is this not in `TrackLocal` trait?

Reply:
I wanted to keep the change small and not have to deal with changing the additional places that implement `TrackLocal`. My thinking is we can always move this into `TrackLocal` in the future